### PR TITLE
conformance: initial commit

### PR DESCRIPTION
**Pull request by rchincha**

### CODE REVIEW

**Comment by rchincha** [comment]
cc: @SteveLasker

**Comment by rchincha** [comment]
Partly addresses issue #22

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+BATS := $(TOOLSDIR)/bin/bats
+BATS_FLAGS = --verbose-run --show-output-of-passing-tests --print-output-on-failure
+
+.PHONY: all
+all: conformance
+
+$(BATS):
+	pushd $(TMPDIR); \
+	git clone https://github.com/bats-core/bats-core.git; \
+	cd bats-core; ./install.sh $(TOOLSDIR); cd ..; \
```
Comment (ktarplee) [inline Makefile]: Nitpick: `cd ..` is not needed since you call `popd` right after that.

Comment (rchincha) [reply Makefile]: Fixed.

**Comment by rchincha** [comment]
This PR intended only as a first step.

**Review by SteveLasker — APPROVED** [review]
LGTM to make forwrad progress and experiment

**Pull request merged**